### DISCUSSION TITLE
next.js/examples/with-firebase/

### DISCUSSION THREAD

**Discussion by alexllao** [vercel/next.js#13400] [Help]
I don't understand the usefulness of this example. It seems very simple but I don't understand what it does.

On the "Index" page I have created a "firebase.app (). Auth (). SignWithEmail ....." with a sample user and it always gives the error that there is no app by default. That could be happening? Thank you.

**Comment by Onibenjo** [comment]
Did you update the environment variables?

**Reply by alexllao** [reply]
Yes!
